deploy/olm-catalog/node-maintenance-operator/0.1.0/node-maintenance-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"kubevirt.io/v1alpha1","kind":"NodeMaintenance","metadata":{"name":"node02"},"spec":{"reason":"Test
+      node maintenance"}}]'
+    capabilities: Basic Install
+  name: node-maintenance-operator.v0.1.0
+  namespace: placeholder
```

Comment:
we may want to fix this

Reply:
the same issue is in ssp operator: https://github.com/MarSik/kubevirt-ssp-operator/blob/master/deploy/olm-catalog/kubevirt-ssp-operator/1.0.0/kubevirt-ssp-operator.v1.0.0.clusterserviceversion.yaml#L8

Reply:
actually, "placeholder" is fine. OLM will set the correct namespace.